lib/tasks/evm.rake

```diff
@@ -104,10 +108,11 @@ namespace :evm do
   task :compile_assets => 'evm:assets:compile'
   namespace :assets do
     desc "Compile assets (clobber and precompile)"
-    task :compile do
+    task :compile do |rake_task|
+      app_prefix = rake_task.name.chomp('evm:assets:compile')
       EvmRakeHelper.with_dummy_database_url_configuration do
-        Rake::Task["assets:clobber"].invoke
-        Rake::Task["assets:precompile"].invoke
+        Rake::Task["#{app_prefix}assets:clobber"].invoke
+        Rake::Task["#{app_prefix}assets:precompile"].invoke
```

Comment:
I'm concerned this will break callers of this that might live outside of this app (like from UI class), though perhaps this is fixing that?

Reply:
> ... though perhaps this is fixing that?

It is, and we actually use this in other places as well, and I just ripped it off (I used the same `app_prefix` variable name, so you should be able to grep in `lib/tasks` to find it).

That said, I was thinking that it might make sense to have a generic helper function for this, but figured it wasn't worth starting on that in the PR, though I could still be convinced if we think it makes sense.

Reply:
I actually found a way to fix a part of this with Rails itself, but never got around to blasting it out...I'd like to pair on it if you have time at some point.  However, I think your prefix code here would still be needed.